docs/api.md

```diff
@@ -281,7 +281,61 @@ curl --request GET \
 
 ### Signin
 
-> TODO
+To signin, send a `POST` request to `https://api.ewea.io/v1/signin`. The following attributes are supported:
+
+<br/>
+
+|    Name     |  Type  |                       Description                       | Required |
+| :---------: | :----: | :-----------------------------------------------------: | :------: |
+|    Email    | String | Registered primary email address used to contact party  |   True   |
```

Comment:
Supported signin attributes are `username` and `password` please assist on update